3542/3/views/sessions/login.erb

```diff
@@ -0,0 +1,26 @@
+<html lang="en">
```

Comment:
use one template engine for views.
(I would prefer haml )
you can convert your erb files using this tool: (https://awsm-tools.com/code/html2haml)